trustkit/src/main/java/com/datatheorem/android/trustkit/config/DomainPinningPolicy.java

```diff
@@ -42,7 +43,12 @@
             throws MalformedURLException {
         // Run some sanity checks on the configuration
         // Check if the hostname seems valid
-        DomainValidator domainValidator = DomainValidator.getInstance(false);
+        DomainValidator domainValidator;
+        if (hostname.equals("127.0.0.1") || hostname.equals("localhost")) {
+            domainValidator = DomainValidator.getInstance(true);
+        } else {
+            domainValidator = DomainValidator.getInstance(false);
```

Comment:
Why not always use `DomainValidator.getInstance(true)` ? It seems like it would result in the exact same behavior or am I missing something?

Reply:
The main is difference is :
https://github.com/datatheorem/TrustKit-Android/blob/3f96ec34d5a1fd8a3bcf24bb7c572dc4fe48170a/trustkit/src/main/java/com/datatheorem/android/trustkit/config/DomainValidator.java#L121-L126

Reply:
@jobot0 probably can be inited in one line
domainValidator = DomainValidator.getInstance(hostname.equals("127.0.0.1") || hostname.equals("localhost"));
but it's matter of style...